File: submissions/sm_034_Venkat/week_3/day_3/session_2/average_divisible.js
function avg(limit, divisor) {
  var limit = limit || 12
  var divisor = divisor || 4
  var count = 0
  var sum = 0
  var avg = 0
  for (i = 1; i <= limit; i++) {
    if (i % divisor == 0) {
      sum += i
      count += 1
    }
  }
  avg = sum / count
  // console.log(sum)
  // console.log(count)
  console.log(avg)
}
avg(29, 3)

File: submissions/sm_115_suyash/week_13/day_4/cooding_1/average_divisible.py
print("Input diviser")
div_num = int(input())
print("Input max limit")
max_limit = int(input())
sum = 0
avg = 0
count = 1
for x in range(0,max_limit,div_num):
    count += 1
    sum += x
else:
    sum += x+div_num

avg = sum / count
print(avg)